src/components/Mentors/EditMentor.js

```diff
@@ -504,6 +506,7 @@ class EditMentor extends Component {
                     id="linkedin"
                     name="linkedin"
                     label="LinkedIn"
+                    placeholder="https://www.linkedin.com/in/your-profile/"
```

Comment:
Placeholder muy largo

Reply:
He incluido un FormHelperText  abajo de cada TextField para incluir este texto.
I have put a FormHelperText to include this text below the TextField.